spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/java/com/alibaba/cloud/nacos/NacosDiscoveryProperties.java

```diff
@@ -279,6 +301,205 @@ public void init() throws Exception {
 		}
 	}
 
+	 class InetUtilsIPV6 implements Closeable {
+		private final ExecutorService executorService;
+
+		private final Log log = LogFactory.getLog(com.alibaba.cloud.nacos.NacosDiscoveryProperties.InetUtilsIPV6.class);
+
+		@Value("${spring.cloud.inetutils.default-hostname}")
+		private String defaultHostname = "localhost";
+
+		@Value("${spring.cloud.inetutils.default-ip-address}")
+		private String defaultIpAddress = "0:0:0:0:0:0:0:1";
+
+		@Value("${spring.cloud.inetutils.ignored-interfaces}")
+		private List<String> ignoredInterfaces = new ArrayList();
+
+		@Value("${spring.util.timeout.sec:${SPRING_UTIL_TIMEOUT_SEC:1}}")
+		private int timeoutSeconds = 1;
+
+		@Value("${spring.cloud.inetutils.use-only-site-local-interfaces}")
+		private boolean useOnlySiteLocalInterfaces = false;
```

Comment:
Whether user can use double stack address register to registry? for example, host has IPv4 and IPv6 address stack at the same time, whether user can register them to registry by configuring the config  information? I think is a reasonable demand. It can help user to immigrate to IPv6 from IPv4 smoothly.

Reply:
I can't understand what means  double stack address register to registry,It has both ipv4 and ipv6?